### PR TITLE
feat(infra-apps): Update Argo CD from 2.7.0 to 2.7.1

### PR DESCRIPTION
**Pull request by hairmare**
# Description

Updates chart to actually include CVE-2023-0464 updates.

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog with an `artifacthub.io/changes` annotation in `Chart.yaml`, check the [example](docs/development.md#Changelog) in the documentation.
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Review by janaurka — APPROVED** [review]
me gusta langusta

File: charts/infra-apps/Chart.yaml [hunk 1]
```diff
@@ -17,12 +17,18 @@ dependencies:
...
+      description: "argo-cd: update chart from 5.30.0 to 5.32.0"
       links:
-        - name: "feat: Upgrade Argo CD to v2.7.0"
-          url: https://github.com/argoproj/argo-helm/pull/1989
+        - name: "feat: allow extraObjects to contain string template"
+          url: https://github.com/argoproj/argo-helm/pull/1993
+        - name: "chore: Update Argo CD to v2.7.1"
+          url: https://github.com/argoproj/argo-helm/pull/1993
+        - name: "feat: Use tini as the ENTRYPOINT"
+          url: https://github.com/argoproj/argo-helm/pull/1993
```
Comment (eyenx) [inline charts/infra-apps/Chart.yaml:32]: All the same PR?

Comment (hairmare) [reply charts/infra-apps/Chart.yaml:32]: ups no, i failed at copypasting :(

Comment (eyenx) [reply charts/infra-apps/Chart.yaml:32]: I saw it when it was in draft but did not say anything as I thought you were still working on it

**Pull request merged**